### ISSUE TITLE
Accessibity improvements needed on the Ontology Page

### ISSUE THREAD

**Issue by miguelvaara** [NatLibFi/Skosmos#1037]
[Make the changes as shown in the illustration]

1) The Ontology Page should offer a possibility to tab through interactive items in a logical order [numbered order in the illustration].

2) The semantic structure of the page should be improved by relevant and comprehensive heading levels [as shown in the illustration]

3) Skip link is a shortcut to important parts of the page (content) and makes the access to content quicker. The skip link helps to avoid unnecessary repetition of static page elements [content area marked in the illustration / the skip link should be located in the beginning of the body tag]
![Ontologiasivu_valmis](https://user-images.githubusercontent.com/50104570/90411345-4edc0a80-e0b4-11ea-88d2-7c7f3a82c93a.jpg)

**Comment by miguelahonen** [comment]
Skip link targets the search options, not the description area

Tab order:
- Language selection
- Search area
- Left side list (tabs) (inside the tabs a natural tab order)
- Description of the ontology or vocabulary
- Top menu

Headers (level H2):
- Search area (hidden header with the text "???")
- Left side area tabs
- Vocabulary description area

Headers (level H3):
- In the Vocabulary description area as in the picture

Check what a screen reader "says" while entering the page. Should be the base uri (or the title?)

**Comment by miguelahonen** [comment]
The skip link should become visible when it receives keyboard focus. Maybe hiding it off screen and then to be positioned on screen when it receives keyboard focus?

**Comment by miguelahonen** [comment]
All the information for a screen reader should be available in Finnish, Swedish and English

**Comment by osma** [comment]
This is relevant for Skosmos 2 only.

We will make a new accessibility review for Skosmos 3, see #1871

Closing this issue as it is no longer relevant for current development.
